src/main/java/com/g4s8/ghman/web/TkApp.java

```diff
@@ -54,7 +56,9 @@
                         new TkFork(
                             new FkParams(
                                 "code", ".+",
-                                new TkGitHubAuthorization(data)
+                                new TkGitHubAuthorization(
+                                    new PgUsers(data), new EnvironmentVariables()
```

Comment:
@olenagerasimova `EnvironmentVariables` must be declared in the constructor to of `TkApp` and passed to all the objects that needs it